project_description/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================
+Project Description
+===================
+
+This module extends the functionality of projects to support setting a
+description for each project and allow you to search by that description.
+
+Usage
+=====
+
+To set a description for a project, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Edit or create one.
+#. You will see a new *Description* tab, with an HTML area to set the project's description.
+
+.. figure:: static/description/image.png
+   :alt: Project form with the description tab
+
+To search for a project by its description, you need to:
+
+#. Go to *Project > Project > Projects*.
+#. Enter any word in the search box.
+#. Select *Search Description*.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/11.0
```

Comment:
@Change2improve IMO Change target branch

Reply:
I am using the new README by fragment system, so I think that the README.rst file will be generated by oca-gen-addon-readme and manual changes will be overwritten.

Should I run this oca-gen-addon-readme command? or it will be run automatically?

Reply:
Yes, But Your target is `11.0` so you can replace `12.0`